draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1932,8 +1932,7 @@ Constants:
   3. the polynomial g(x) - Z is irreducible over F, and
   4. g(B / (Z * A)) is square in F.
 
-Sign of y: Inputs u and -u give the same x-coordinate.
-Thus, we set sgn0(y) == sgn0(u).
+Sign of y: Inputs u and -u give the same x-coordinate, so sgn0(y) == sgn0(u).
```

Comment:
Same as prior comment---can we keep explicit that this is a decision?

Reply:
Same!